### PR TITLE
TC: Add nightly documentation promotion for release

### PR DESCRIPTION
**Pull request by cobexer**

### CODE REVIEW

File: .teamcity/src/main/kotlin/promotion/PromotionProject.kt [hunk 1]
```diff
@@ -30,6 +30,10 @@ class PromotionProject(branch: VersionedSettingsBranch) : Project({
         buildType(PublishFinalRelease(branch))
     }
 
+    if (branch.isRelease || branch.isExperimental) {
```
[the remark this answers is not in the document]
Comment (cobexer) [reply .teamcity/src/main/kotlin/promotion/PromotionProject.kt:33]: We don't plan to do that as nightly is available as https://docs.gradle.org/nightly/ already anyway.

**Comment by cobexer** [comment]
@bot-gradle merge

**Review by oleg-nenashev — APPROVED** [review]
Definitely +1. IIUC we will get much faster docs delivery with this change

**Comment by cobexer** [comment]
The relevant job: https://builds.gradle.org/buildConfiguration/Gradle_Release_Promotion_NightlyDocumentation?branch=&mode=builds

**Pull request merged**